resources/resources.md

```diff
@@ -0,0 +1,16 @@
+---
+nav_order: 121
+has_children: true
+title: Resources
+description: Machine translation resources
+---
+
+### Calls for papers
+
+| Publication | Organisers | Deadline | Type |
+| --- | --- | --- | --- |
+| [The impact of Machine Translation in the Audiovisual Translation environment](https://lans-tts.uantwerpen.be/index.php/LANS-TTS/announcement/view/21) | Universitat Jaume I | April 01, 2022 | Scientific publication |
```

Comment:
I thought we were doing dates without the leading *0*?  And maybe without the comma?

Reply:
Actually, we agreed on having the leading 0. Every event has the leading 0 as well.